app/services/analytics_events.rb

```diff
@@ -1011,6 +1011,12 @@ def idv_gpo_confirm_start_over_visited
     track_event('IdV: gpo confirm start over visited')
   end
 
+  # A GPO reminder email was sent to the user
+  # @param [String] user_id UUID of user who we sent a reminder to
+  def idv_gpo_reminder_email_sent(user_id:, **_extra)
+    track_event('IdV: gpo reminder email sent', user_id: user_id)
```

Comment:
the goal of `**extra` is to pass it in to the underlying track_event
```suggestion
  def idv_gpo_reminder_email_sent(user_id:, **extra)
    track_event('IdV: gpo reminder email sent', user_id: user_id, **extra)
```

Reply:
Fixed.